FinniversKit/Sources/Components/SelectionListView/SelectionListView.swift

```diff
@@ -15,6 +15,7 @@ public class SelectionListView: UIView {
 
     public let presentation: Presentation
     public weak var delegate: SelectionListViewDelegate?
+    let isSeparatedListElementDesign: Bool
```

Comment:
Open to suggestions for a better name.

Reply:
What about baking this into configurable spacing between elements? If there is no spacing then the original design is used. Maybe not explicit enough?

Reply:
I prefer being more explicit with this, but a separate parameter/flag might be overkill

Reply:
Separate types of lists is also an option, but the differences are not enough yet for that I think.